guides/common/modules/proc_updating-disconnected-server.adoc

```diff
@@ -0,0 +1,163 @@
+[id="Updating-Disconnected-{project-context}_{context}"]
+= Updating a Disconnected {ProjectServer}
+
+Use this procedure to update your air-gapped {Project} setup where the connected {ProjectServer}, which synchronizes content from CDN, is air gapped from a disconnected {ProjectServer}, to the next minor version.
```

Comment:
```suggestion
Update your air-gapped {Project} setup where the connected {ProjectServer}, which synchronizes content from CDN, is air gapped from a disconnected {ProjectServer}, to the next minor version.
```
We should stop using this self-referential content as per IBM style guide. Please, rewrite also the other occurences in this PR.

Reply:
unrelated to your PR: I just did a quick `grep` for "air-gapped" and "disconnected". I'm wondering if _solely_ using "disconnected" is a good idea?

Reply:
I would also go with `Disconnected` in place of `Air-Gapped` but will rely on your views. We can have a separate PR for the same.

Reply:
Reviewed at other places as well @Lennonka :+1: